drv/mock-gimlet-seq-server/src/main.rs

```diff
@@ -58,6 +58,7 @@ impl idl::InOrderSequencerImpl for ServerImpl {
         &mut self,
         _: &RecvMessage,
         state: PowerState,
+        _: StateChangeReason,
```

Comment:
Perhaps we should add a ringbuffer to the mock sequencer implementation on Gimletlet, to aid in debugging in its callers?

Reply:
@jgallagher yeah, that's why I didn't do it in this branch, it seemed straightforward enough to add later if someone actually needed it for some reason.